fonttools/genft.cpp

```diff
@@ -81,6 +81,9 @@ QMap<QString, QString> nmap {
       { "rests.5",                  "rest32nd" },
       { "rests.6",                  "rest64th" },
       { "rests.7",                  "rest128th" },
+//    { "rests.8",                  "rest256th" },  // Glyphs currently don't exist in Emmentaler
```

Comment:
I'm wondering if `genft` is actually doing anything at all at this point. It seems like it was used previously when converting the original Emmantaler to the MuseScore font, but since the two have forked I don't really understand what this code does anymore. Do we still need it at all?